DOCS-2449: update IBAN testing details for bank transfer (#1798)
MultiSafepay/docs: content/payments/integrations/testing/test-payment-details.md

{{< /details >}}

{{< details title="Bank Transfer" >}}

Test credentials:

> - [API key](/tools/multisafepay-control/get-your-api-key/)

Reviewer: ```suggestion
- [API key](/account/site-id-api-key-secure-code/)
```

> **Test a Bank Transfer**

Reviewer: ```suggestion
**Test a Bank Transfer order**
```

> 1. To test a Bank Transfer, send a [redirect](/api/#bank-transfer---redirect) API request.

Reviewer: ```suggestion
1. To test a Bank Transfer order, send a [redirect](/api/#bank-transfer---redirect) API request.
```

2. Open the payment link. 
3. In the **Your bank account** field, enter an IBAN. 

> 4. In the **Bank's country** list, select a country, and then click **Confirm**.

Reviewer: ```suggestion
4. From the **Bank's country** list, select a country, and then click **Confirm**.
```

Use the following IBANs to test different transaction statuses.

| IBAN | Status    | Description              |
| ---------| --------- | ------------------------ |
| NL87ABNA0000000001| **Initialized**/ **Completed** | Transaction is initialized. After 2 minutes, this changes to **Completed**. |
| NL87ABNA0000000002| **Initialized**/ **Expired** | Transaction is initialized. After 2 minutes, this changes to **Expired**. |
| NL87ABNA0000000004| **Initialized**/ **Declined** | Transaction is initialized. After 2 minutes, this changes to **Declined**. |
| Any other IBAN | **Initialized**/ **Expired** | Transaction is initialized. After 5 minutes, this changes to **Expired**. |

**Test cancelling an order**

> To test cancelling an order, either:

Reviewer: Might need a blank line to render the bullets correctly in live?

- Send an [update an order](/api/#update-an-order) API request with status **Cancelled**, or 

> - In your MultiSafepay test account, go to **Order Summary**, and then click **Cancel**.  

Reviewer: ```suggestion
- In your MultiSafepay test account, go to **Order summary**, and then click **Cancel**.  
```

The transaction status changes to **Void**.

**Test refunding an order**

To test refunding an order:

1. Create an order using IBAN: NL87ABNA0000000001. 

> 2. Wait until the transaction status is **Completed**.

Reviewer: ```suggestion
2. Wait until the transaction status changes to **Completed**.
```

> 3. In your MultiSafepay test account, go to **Order Summary**, and then click **Refund order**.

Reviewer: ```suggestion
3. In your MultiSafepay test account, go to **Order summary**, and then click **Refund order**.
```

4. Under **Refund**, in the **Account holder name** field, enter the account holder name of the account you want to refund to. 
5. In the **IBAN** field, enter the IBAN of the account you want to refund to.
6. In the **Reason/Description** field, enter a reason for the refund. 
7. In the **Amount** field, enter the amount to refund, and then click **Continue**.
8. Under **Refund confirmation**, check that the description and amount are correct, and then click **Confirm**.
  {{< br >}} A new order is created for the refund. The order status for the refund changes to **Reserved**.
9. Under **Related transactions**, select the **ID** of the refund order.
10. Under **Order summary**, click **Accept**.
11. In the **Add transaction comment** field, add a comment, and then click **Add**.
  The order status changes to **Completed**.

---

> **Note**: You can't test the following:

Reviewer: ```suggestion
**Note:** You can't test the following:
```

Reviewer: Our style is to bold the colon as well :)
